### PR TITLE
Updated comfirmation modal to show executed order details

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

[Second part](https://www.notion.so/cownation/Show-Me-the-Money-hackathon-0a1b1b3e6a1b4d4eb9a46f96136a1865?pvs=4#638f3d64c891445bb3c04e7c046374b7) of the surplus saga

This PR updates the confirmation modal to show order summary and surplus once the order is executed
Actually once the order is executed it will first show confirmation text as before and then after 4s it will show this new content

New content
Screenshot 2023-04-12 at 15 40 39

### To test
1. post an order
2. when confirmation modal opens don't close it
3. wait until the order is executed
4. after 4s you should see this new content

### CODE REVIEW

File: src/cow-react/modules/wallet/web3-react/connection/trust.tsx [hunk 1]
```diff
@@ -22,11 +22,11 @@ const BASE_PROPS = {
...
-  new InjectedWallet({
-    actions,
-    walletUrl: WALLET_LINK,
-    searchKeywords: ['isTrust', 'isTrustWallet'],
-  })
+    new InjectedWallet({
+      actions,
+      walletUrl: WALLET_LINK,
+      searchKeywords: ['isTrust', 'isTrustWallet'],
+    })
```
Comment (anxolin) [inline src/cow-react/modules/wallet/web3-react/connection/trust.tsx:29]: what's with these changes? by now i saw them in 3 PRs

File: src/custom/components/OrderProgressBar/index.tsx [hunk 2]
```diff
@@ -32,21 +32,29 @@ import { getExplorerOrderLink } from 'utils/explorer'
...
 import { CancelButton } from '@cow/common/pure/CancelButton'
+import { getExecutedSummaryData } from '@src/custom/state/orders/helpers'
+import { Order } from 'state/orders/actions'
+import styled from 'styled-components/macro'
+import { DisplayLink } from '../TransactionConfirmationModal'
+import { TokenAmount } from '@cow/common/pure/TokenAmount'
 
 const REFRESH_INTERVAL_MS = 200
 const COW_STATE_SECONDS = 30
+const SHOW_CONFIRMED_MS = 4000
```
Comment (anxolin) [inline src/custom/components/OrderProgressBar/index.tsx:43]: can you use ms lib?

File: src/custom/components/OrderProgressBar/index.tsx [hunk 3]
```diff
@@ -95,9 +103,19 @@ export function OrderProgressBar(props: OrderProgressBarProps) {
...
       setPercentage(100)
+
+      timeout = setTimeout(() => {
+        setShowDetails(true)
+      }, SHOW_CONFIRMED_MS)
+    } else {
+      setShowDetails(false)
     }
+
+    return () => clearTimeout(timeout)
```
Comment (anxolin) [inline src/custom/components/OrderProgressBar/index.tsx:118]: timeout can be undefined, is it ok to use it like this?

File: src/custom/components/OrderProgressBar/index.tsx [hunk 4]
```diff
@@ -342,3 +362,74 @@ function useGetProgressBarInfo({
     isPending: orderIsPending,
   }
 }
+
+const ExecutedWrapper = styled.div`
```
Comment (anxolin) [inline src/custom/components/OrderProgressBar/index.tsx:366]: can you use this oportunities to make dumb components and ideally cosmos?

**Comment by MindyCoW** [comment]
Also here, we should write "You received a surplus of X COW on this trade!"

File: src/cow-react/modules/wallet/web3-react/connection/trust.tsx (hunk 1 above)
Comment (nenadV91) [reply src/cow-react/modules/wallet/web3-react/connection/trust.tsx]: Its the Prettier autocommit 😄 
Then I forget to pull before I do changes and then I force push :smile

File: src/custom/components/OrderProgressBar/index.tsx (hunk 2 above)
Comment (nenadV91) [reply src/custom/components/OrderProgressBar/index.tsx:43]: Updated

File: src/custom/components/OrderProgressBar/index.tsx (hunk 3 above)
Comment (nenadV91) [reply src/custom/components/OrderProgressBar/index.tsx:118]: `clearTimeout` takes undefined as a param also, so I think yes.

**Comment by nenadV91** [comment]
@elena-zh Good suggestion, fixed
Screenshot 2023-04-13 at 11 45 24

@MindyCoW Updated text

**Comment by elena-zh** [comment]
Hey @nenadV91 , something is wrong with Infura in the current PR: 
image
image

**Comment by nenadV91** [comment]
Same comment as [here](https://github.com/cowprotocol/cowswap/pull/2289#issuecomment-1506999778)

**Pull request merged**